### ISSUE TITLE
Search Results and Pagination - 3.3.2 Labels or Instructions - 1 missing form label, 1 select missing label and 1 empty form label are present

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9713]
1 missing form label, 1 select missing label and 1 empty form label on the Search field. Missing or empty form labels may affect screen reader interpretation of form fields meaning the function/purpose of the form may not be communicated to the user.

**Linked work item:** 
#9521 

**Expected Result:**
No missing, empty or select missing form labels should be present.

**Actual Result:**
1 missing form label, 1 select missing label and 1 empty form label are present, all on the Search field.

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/651457bb-1d25-4766-b56b-a90281a009c9)
![Image](https://github.com/archesproject/arches/assets/77161085/e41c1618-6e42-4f56-ae56-f0b144eaeee5)
![Image](https://github.com/archesproject/arches/assets/77161085/21757abc-587d-4b15-b33c-bc0f490d4192)

**Comment by johnatawnclementawn** [comment]
This issue is dependent on our select2/chosen replacement work

**Comment by apeters** [comment]
@imogenroberts this should be fixed in the latest, can you recheck?

**Comment by imogenroberts** [comment]
RETEST: Missing, select missing, and empty form labels are all resolved and no errors are present, on this or nay other component on the page.

**Evidence:**


![Image](https://github.com/archesproject/arches/assets/77161085/a0501361-c5c4-49ba-a0dd-fe6325cab3e3)

**Comment by apeters** [comment]
Yay!
